server-common/src/main/java/org/apache/kafka/server/common/TestFeatureVersion.java

```diff
@@ -23,8 +23,8 @@ public enum TestFeatureVersion implements FeatureVersion {
 
     // TEST_1 released right before MV 3.7-IVO was released, and it has no dependencies
     TEST_1(1, MetadataVersion.IBP_3_7_IV0, Collections.emptyMap()),
-    // TEST_2 released right before MV 3.8-IVO was released, and it depends on this metadata version
-    TEST_2(2, MetadataVersion.IBP_3_8_IV0, Collections.singletonMap(MetadataVersion.FEATURE_NAME, MetadataVersion.IBP_3_8_IV0.featureLevel()));
+    // TEST_2 released right before MV 4.0-IVO was released, and it depends on this metadata version
```

Comment:
4.0-IVO => 4.0-IV0 This is an existing issue. The trailing number should be 0, not an O.

Ditto for 3.7-IVO above.